open3d_slam/include/open3d_slam/cloud_processing.hpp

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
is this file needed? can it not be made a static method of the Registration base class or so?

Reply:
The registration base class could be great but I also see value on keeping a `cloud processor` file which might be extended with box filters, retrievers, feature extractors

Reply:
I can probably remove the separate file. For now we don't need it.